src/QsCompiler/BondSchemas/QirExecutionWrapper/QirExecutionWrapper.bond

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import "EntryPoint/EntryPoint.bond";
+namespace Microsoft.Quantum.QsCompiler.BondSchemas.QirExecutionWrapper;
+
+struct Bytecode
+{
+    5: blob Data;
+}
+
+struct QirExecutionWrapper
+{
+    5: Microsoft.Quantum.QsCompiler.BondSchemas.EntryPoint.EntryPointOperation EntryPoint;
+    10: bonded<Bytecode> QirBytes;
```

Comment:
```suggestion
    10: bonded<Bytecode> QirBytecode;
```

Reply:
This both a rename and a reference to [this comment](https://github.com/microsoft/qsharp-compiler/pull/945#discussion_r607437511).